crates/astria-sequencer/src/mempool/benchmarks.rs

```diff
@@ -325,8 +328,11 @@ fn run_maintenance_tx_recosting<T: MempoolSize>(bencher: divan::Bencher) {
         .unwrap();
     // Set the new nonce so that the entire `REMOVAL_CACHE_SIZE` entries in the
     // `comet_bft_removal_cache` are filled (assuming this test case has enough txs).
-    // allow: this is test-only code, using small values, and where the result is not critical.
-    #[allow(clippy::arithmetic_side_effects, clippy::cast_possible_truncation)]
+    #[expect(
+        clippy::arithmetic_side_effects,
+        clippy::cast_possible_truncation,
+        reason = "this is test-only code, using small values, and where the result is not critical"
+    )]
```

Comment:
Same here .. the `allow` is so wordy, it'd be easier to just use checked arithmetic.